wrappers/Julia/CoolProp.jl

```diff
@@ -220,6 +220,49 @@ function AbstractState_update(handle::Clong,input_pair::Clong,value1::Number,val
   end
   return nothing
 end
+function AbstractState_update(handle::Clong,input_pair::AbstractString,value1::Real,value2::Real)
+  AbstractState_update(handle::Clong,get_input_pair_index(input_pair),value1::Real,value2::Real)
+  return nothing
+end
```

Comment:
Overloading the function to be able to give the input pair in the form of a string (calling internally get_input_pair_index).
This allows to write code faster when only one update with this pair is made.